sympy/functions/elementary/complexes.py

```diff
@@ -4,7 +4,7 @@
 from sympy.core.compatibility import u
 from sympy.core.exprtools import factor_terms
 from sympy.core.function import (Function, Derivative, ArgumentIndexError,
-    AppliedUndef)
+                                 AppliedUndef)
```

Comment:
You said "not in this case".
And yes I read PEP8, please stop assuming I'm retarded.